src/js/components/uploadDetachedFiles/UploadDetachedFilesPage.jsx

```diff
@@ -16,6 +16,8 @@ import UploadDetachedFileValidation from './UploadDetachedFileValidation.jsx';
 
```

Comment:
Overall question for this file, why are we even bothering with a `showMeta` state? The only time it comes into play other than for setting itself is deciding which to display. Can't you just base it off the params? Or does it not re-render if that's the case?